backend/hct_mis_api/apps/registration_datahub/validators.py

```diff
@@ -1352,7 +1352,7 @@ def _get_field_type_error(self, field: str, value: Any, attachments: list) -> Un
 
             complex_types: Dict[str, Callable] = {
                 "GEOPOINT": self.geopoint_validator,
-                "IMAGE": self.image_validator,
+                # "IMAGE": self.image_validator,  # comment for now as hotfix AB#209023 for skip image validation
```

Comment:
@pavlo-mk this should depend of the form flag.
Pls also change default to check for the flag

Reply:
updated, now it will based on `KoboImportData. pull_pictures` (bool)